docs/pipelines/tasks/utility/download-package.md

```diff
@@ -20,24 +20,25 @@ monikerRange: 'azure-devops'
 Use this task in a build or release pipeline to download a package from a package management feed in Azure Artifacts or TFS.
 Requires the Package Management extension.
 
-> [!NOTE]
-> This task currently only supports downloading NuGet packages.
-
 ::: moniker range="> tfs-2018"
 
 ## YAML snippet
 
-[!INCLUDE [temp](../_shared/yaml/DownloadPackageV0.md)]
+[!INCLUDE [temp](../_shared/yaml/DownloadPackageV1.md)]
 
 ::: moniker-end
 
 ## Arguments
 
 <table><thead><tr><th>Argument</th><th>Description</th></tr></thead>
+<tr><td>Package type</td><td>(Required) Select the type of package to download</td></tr>
 <tr><td>Feed</td><td>(Required) Select the package source</td></tr>
```

Comment:
Did you want to update this to say something like:
"ID of the feed that contains the package. For project-scoped feeds, the format is project ID/feedID. See our [Q&As](#Q&A) below for information on how to get a feed or project ID, or information on using project and feed name instead."

Reply:
Fixed